### ISSUE TITLE
Implementing V2 gateway label icons on V1 checkouts

### ISSUE THREAD

**Issue by FangedParakeet** [Automattic/woocommerce-payments#9826]
### Description

Stripe has now added a feature to remove the card icon animations on the payment element (see p1732125470634879-slack-C043JN7AJHY). We would now like to continue with replacing the generic card icon added to the payment gateway label in #9483 with the V2 card icons component. 

![Card gateway icons](https://cdn-std.droplr.net/files/acc_1104206/Utt8g6)
_This issue concerns `1` and `2` above_

We would like to add these icons to both shortcode and blocks checkouts--this can possibly handled in separate issues, if either implementation is deemed individually complex enough.

**Note that while this Stripe feature is currently gated to our dev account, we will need to plan how this will be rolled out to all stores. Consequently, while we can complete this integration, we may want to delay on merging and closing this issue, until this rollout plan has been confirmed and aligned.**

### Acceptance criteria

* Ensure payment element icons animation is no longer present at checkout.
   * May need to test on gated dev account temporarily, see p1732125470634879-slack-C043JN7AJHY.
* Implement V2 card icon label badge with tooltip on payment gateway label.
   * This will need to be present on both shortcode and blocks checkouts, though we may decide to divide this into separate issues/PRs.

### Designs

[Figma](https://www.figma.com/design/g1bWnb6JBO3X3bGrgJnRJe/Agnostic-Payment-Component?node-id=154-7168&node-type=frame&t=H5SCEMfK24FQfhgB-0)
pfHfG4-gT-p2#comment-201

### Dev notes
Related to #9331 and #9475.

### Additional context

p1732125470634879-slack-C043JN7AJHY
pfHfG4-gT-p2#comment-201

**Comment by mdmoore** [comment]
I bumped the estimate on this up to a 3 as we're waiting on Stripe to disable card logos in our dev environments. This is completed for the checkout block pending testing against the icon removal. Classic checkout is being tracked in #10026 since it's a separate implementation, but the I'll likely include the code with [this PR](https://github.com/Automattic/woocommerce-payments/pull/10005).

**Comment by pierorocca** [comment]
Noting that the label is to be changed from 'Credit card / debit card' to simply 'Cards' biasing to simplicity for mobile and since the Test mode badge + icons will take up most of the space.

**Comment by mdmoore** [comment]
@pierorocca My understanding is that, because the card icon animations feature is all or nothing from an accounts perspective, we'll want to time the release of this with Stripe. If we merge the feature this week, the switch would need to occur on February 5 with the next WooPayments release. Of course, we can hold off to sync on any timeline needed. Just let us know when you have a date and we can proceed accordingly.

**Comment by pierorocca** [comment]
> @pierorocca My understanding is that, because the card icon animations feature is all or nothing from an accounts perspective, we'll want to time the release of this with Stripe. If we merge the feature this week, the switch would need to occur on February 5 with the next WooPayments release. Of course, we can hold off to sync on any timeline needed. Just let us know when you have a date and we can proceed accordingly.

Yeah this is a challenge @mdmoore  as it's all or nothing despite my explicit request to have it be an API property. I'm ok with a short window of icons in two places. We had that for the longest time with that 4 logo approach. What's more problematic is the opposite scenario where there are no icons - none in the radio button and none in the card number field. This would be the case for merchants on any previous version of WooPayments between now and the 4 logo approach.

Is there a way to stagger this somehow so merchants R-1 wouldn't be impacted?

**Comment by mdmoore** [comment]
>Is there a way to stagger this somehow so merchants R-1 wouldn't be impacted?

Unfortunately, I don't see a solution for merchants that don't yet have the new feature. Having icons in both places for a bit is the best solution IMO. If that sounds good, we can release this with 8.9.0 and hold off on removing the card field icons.
